HelpSource/Classes/GrayNoise.schelp

```diff
@@ -24,13 +37,29 @@ This value will be added to the output.
 Examples::
 
 code::
-
 (
 SynthDef("help-GrayNoise", { |out = 0|
 	Out.ar(out,
 		GrayNoise.ar(0.1)
 	)
 }).play;
 )
+::
+
+subsection:: Comparison with a version of inverse-A-weighted noise
+code::
+{ GrayNoise.ar * 0.1 }.freqscope
```

Comment:
```supercollider
{ GrayNoise.ar * -20.dbamp }.freqscope
```

Reply:
And maybe use -20.dbamp in the main example?

Reply:
Thanks, Done!

Reply:
I also suggested to change the main example above, if possible: `GrayNoise.ar(0.1)` -> `GrayNoise.ar(-20.dbamp)`.

Reply:
Sorry, done in https://github.com/supercollider/supercollider/pull/6735/commits/cc09abfaf478c28d33ca20ee20b1032630283980